financialaid/views_test.py

```diff
@@ -82,13 +135,78 @@ def test_income_validation_user_not_enrolled(self):
         """
         self.program_enrollment.user = self.profile2.user
         self.program_enrollment.save()
-        resp = self.client.post(self.url, self.data)
+        resp = self.client.post(self.income_validation_url, self.data)
         assert resp.status_code == HTTP_400_BAD_REQUEST
 
     def test_income_validation_currency_not_usd(self):
         """
-        Tests IncomeValidationView post  Only takes USD
+        Tests IncomeValidationView post; only takes USD
         """
         self.data["original_currency"] = "NOTUSD"
-        resp = self.client.post(self.url, self.data)
+        resp = self.client.post(self.income_validation_url, self.data)
         assert resp.status_code == HTTP_400_BAD_REQUEST
+
+    def test_review_financial_aid_view_not_allowed(self):
+        """
+        Tests ReviewFinancialAidView that are not allowed
```

Comment:
Can you test for the case where `CoursePrice` with `is_valid=True` does not exist within the program?